.travis.yml

```diff
@@ -42,8 +42,7 @@ script:
     - pytest --cov
     - if [[ $TRAVIS_OS_NAME == "linux" ]] && [[ $PYTHONVER == 3.6 ]]; then
           coverage run --source=exa setup.py test;
-          rm -rf docs/source/*.txt;
-          SPHINX_APIDOC_OPTIONS=members,undoc-members,show-inheritance sphinx-apidoc -eM -s txt -o docs/source/ exa *test*;
+          ./make-docs.sh;
           travis-sphinx build;

```

Comment:
Doesn't this just do something similar to using make html. I think the shell script should mainly be to make it easier for a user to build the docs with the same settings as travis.

Reply:
I think you're right that this should be reverted to the original.